# Examen 2

## Situación 1

Un vuelo transatlántico es el vuelo de una aeronave a través del Océano Atlántico, empezando desde el continente americano y terminando en Europa o África.

<img src="map.png" align="middle" width="800">

**1. Si deseamos realizar un viaje transatlántico desde Estados Unidos a Rusia, pensando que el vuelo toma una trayectoria en línea recta, ¿a cuántos kilómetros se encuentran estos dos países? (Considera que el mapa está a una escala de $3:6011\ \rm{km}$.)**

Primero notemos que las coordenadas de los puntos que nos dan son:
* $\rm{A}(17,2)$
* $\rm{B}(6,3)$
* $\rm{EU}(4,7)$
* $\rm{R}(11,8)$

La distancia entre Estados Unidos (EU) y Rusia (R) es

$$ d=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

$$ =\sqrt{[(4)-(11)]^2+[(7)-(8)]^2} $$

$$ =\sqrt{(-7)^2+(-1)^2} $$

$$ =\sqrt{49+1} $$

$$ =\sqrt{50} $$

Entonces EU y R se encuentran  $\sqrt{50}=5\sqrt{2}\approx7.071$ unidades de distancia (sobre el mapa).

Tenemos que hacer regla de tres
\begin{align*}
3\ \rm{unidades} &\ \leftarrow\ 6011\ \rm{km}\\
&\ \nearrow\\
\sqrt{50}\ \rm{unidades}&
\end{align*}

Entonces, al distancia real que tendrá que viajar el avión es

$$\sqrt{50}\cdot6011\div3\ \rm{km}$$

ó

$$\dfrac{6011\sqrt{50}}{3}\ \rm{km}$$

A) $\dfrac{3\sqrt{18}}{6011}$

B) $\dfrac{3\sqrt{50}}{6011}$

C) $\dfrac{6011\sqrt{18}}{3}$

D) $\dfrac{6011\sqrt{50}}{3}$

**Solución:** D)

**2. Si un avión vuela en línea recta desde Estados Unidos a Rusia, ¿cuál es la ecuación general de la recta que modela ese viaje?**

* $\rm{EU}(4,7)$
* $\rm{R}(11,8)$

Primero obtenemos la pendiente:

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ =\dfrac{(8)-(7)}{(11)-(4)} $$

$$ =\dfrac{1}{7} $$

Esperamos que la ecuación de la recta tenga la forma:

$$ y=mx+b $$

Sustituimos el valor calculado de $m$:

$$ y=\frac{1}{7}x+b $$

Despejamos $b$:

$$ b=y-\frac{1}{7}x $$

Sustituimos las coordenadas de alguno de los dos puntos; por ejemplo, $\rm{EU}(4,7)$ :

$$ b=(7)-\frac{1}{7}(4) $$

$$ =7-\frac{4}{7} $$

$$ =\dfrac{49}{7}-\frac{4}{7} $$

$$ =\dfrac{45}{7} $$

Entonces, la ecuación de la recta en su forma ordenada-pendiente es:

$$ y=\dfrac{1}{7}x+\frac{45}{7} $$

Como la queremos en su forma general pasamos todo al mismo lado de la ecuación:

$$ -\dfrac{1}{7}x+y-\frac{45}{7}=0 $$

Vemos que ninguna de las opciones tiene fracciones, excepto la D),pero esa no puede ser porque está en la forma ordenada-pendiente, además de que no coincide con nuestro resultado. Entonces, nos deshacemos de del denominado multiplicando toda la ecuación anterior por $7$.

$$ 7\cdot\left(-\dfrac{1}{7}x+y-\frac{45}{7}=0\right) $$

$$ -x+7y-45=0 $$

Lo que acabamos de obtener se parece bastante al inciso C), pero tiene el signo del 45 al revés. Entonces, esa no es la respuesta correcta.

Puede ser que trate de un múltiplo de la ecuación que obtuvimos. Por ejemplo, podemos multiplicar la ecuación anterior por $2$.

$$ -2x+14y-90=0 $$

Como queremos que el $2x$ sea positivo, pasamos todo para el lado derecho de la ecuación.

$$ 0=2x-14y+90 $$

(Lo mismo se puede lograr si se multiplica toda la ecuación por $-1$. **Recuerden:** esto de multiplicar o dividir una ecuación por algo se puede hacer siempre y cuando haga dicha operación a ambos lados de la ecuación.)

Como lo que obtuvimos no coincide con la opción A), seguramente se trata de la $C)$. Comprobamos esto:

Podemos multiplicar la ecuación
$$ -x+7y-45=0 $$
por $3$.

$$ -3x+21y-135=0 $$

Como queremos que el $3x$ sea positivo, pasamos todo para el lado derecho de la ecuación.

$$ 0=3x-21y+135=0 $$

A) $2x-14y+6=0$

B) $3x-21y+135=0$

C) $-x+7y+45=0$

D) $y=\dfrac{1}{7}x-\dfrac{45}{7}$

**Solución:** B)

Como extra quisiera decirles que, otra forma de resolver el ejercicio es sustituyendo las coordenadas de los puntos en todas las ecuaciones hasta ver cuál sí se cumple. Por ejemplo:

Sustitución de $\rm{EU}(4,7)$ en la opción A)

$$2x-14y+6=2(4)-14(7)+6=8-98+6=-84\neq0$$

Sustitución de $\rm{R}(11,8)$ en la opción A)

$$2x-14y+6=2(11)-14(8)+6=22-112+6=-84\neq0$$

Ninguna de las dos ecuaciones se cumplen. Por lo tanto, el inciso correcto **NO** es el A)

Sustitución de $\rm{EU}(4,7)$ en la opción B)

$$3x-21y+135=3(4)-21(7)+135=12-147+135=0$$

Sustitución de $\rm{R}(11,8)$ en la opción B)

$$3x-21y+135=3(11)-21(8)+135=33-168+135=0$$

Ambas ecuaciones se cumplen. Por lo tanto, el inciso correcto **SÍ** es el B)

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la recta correcta:

<img src="ej2.png" align="middle" width="1000">

**3. ¿Cuál es la ecuación de la recta perpendicular a la recta que modela el viaje entre Estados Unidos y Rusia, y que pasa por Australia?**

Como ya sabemos la pendiente de la recta que pasa por Estados Unidos y Rusia ($m=\dfrac{1}{7}$), la pendiente perpendicular tiene que ser voltear la fracción y cambiar el signo, es decir,

$$ -\dfrac{7}{1} $$

Entonces, la pendiente de nuestra recta es $m^\prime=-7$.

Si esto no queda tan claro pueden pensarlo con la fórmula

$$ mm^\prime=-1 $$

Es esta despejamos la nueva pendiente y sustituimos la pendiente original.

Esperamos que la ecuación de la recta perpendicular a la anterior tenga la forma:

$$ y=m^\prime x+b^\prime $$

Sustituimos el valor calculado de $m^\prime$:

$$ y=-7x+b^\prime $$

Despejamos $b^\prime$:

$$ b^\prime=y+7x $$

Sustituimos las coordenadas de Australia
$$\rm{A}(17,2)$$
ya que sabemos que tiene que pasar por ahí:

$$ b=(2)+7(17) $$

$$ =2+119 $$

$$ =121 $$

Entonces, la ecuación de la recta perpendicular en su forma ordenada-pendiente es:

$$ y=-7x+121 $$

Como la queremos en su forma general pasamos todo al mismo lado de la ecuación:

$$ 7x+y-121=0 $$

A) $7x+y-117=0$

B) $7x-y-117=0$

C) $7x-y-121=0$

D) $7x+y-121=0$

**Solución:** D)

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la recta correcta:

<img src="ej3.png" align="middle" width="1000">

**4. Si el avión vuela en línea recta desde Estados Unidos hasta Rusia y justo aquí gira para ir hacia Australia en línea recta, ¿cuál es el ángulo que tiene girar?**

Lo que queremos es que el vuelo sea de la siguiente forma:

<img src="vuelo_escala.png" align="middle" width="800">

Y lo que nos interesa es encontrar el ángulo que gira el avión al momento de estar sobre Rusia. Por el dibujo esperamos que el ángulo sea mayor que $90^\circ$. 

Podría haber varia maneras de hacer esto. Lo más fácil es obtener ángulo entre rectas. Nos conviene encontrar la pendiente que forma Rusia con Australia:

* $\rm{R}(11,8)$
* $\rm{A}(17,2)$

Primero obtenemos la pendiente:

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ =\dfrac{(2)-(8)}{(17)-(11)} $$

$$ =\dfrac{-6}{6} $$

$$ =-1 $$

Como ya sabemos que la pendiente entre Estados Unidos y Rusia es $m=\dfrac{1}{7}$, podemos usar la fórmula de ángulo entre pendientes:

$$ \theta=\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

$$ =\tan^{-1}\,\left[\dfrac{(-1)-\left(\dfrac{1}{7}\right)}{1+(-1)\left(\dfrac{1}{7}\right)}\right] $$

In [1]:
((-1)-(1/7))/(1+(-1)(1/7))

-1.3333333333333333

In [2]:
atand(-1.3333333333333333)

-53.13010235415598

$$ \theta\approx-53.13^\circ $$

Recuerden que ignoramos el signo. Como este ángulo es menor que $90^\circ$, queremos el suplementario de éste.

In [3]:
180-53.13010235415598

126.86989764584402

Ahora sí obtuvimos un ángulo que es mayor que $90^\circ$.

A) $143.13^\circ$

B) $90^\circ$

C) $135^\circ$

D) $126.87^\circ$

**Solución:** D)

**5. Al tomar un vuelo que va a varias partes del mundo y pasa por Brasil, un avión viaja en la forma de la ecuación**

$$ \frac{7x^2+5y^2}{5xy+31x+41y-250}=2 $$

**¿Qué tipo de cónica representa la ecuación anterior?**

¿Variables en el denominador? Hay que pasarlas para el otro lado.

$$ 7x^2+5y^2=2(5xy+31x+41y-250) $$

Ahora pasamos todo al mismo lado para tener la ecuación en su forma general:

$$ 7x^2+5y^2=10xy+62x+82y-500 $$

$$ 7x^2-10xy+5y^2-62x-82y+500=0 $$

Comparamos la ecuación que obtuvimos con la ecuación general de segundo grado:

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

Entonces los coeficientes son:

$A=7$, $B=-10$, $C=5$, $D=-62$, $E=-82$, $F=500$

Realmente sólo me importan los primeros tres coeficientes.

Como $B$ no es cero, *invocamos el indicador* (el interior de la raíz cuadrada en *Le Chicharronera*):

$$ I=B^2-4AC $$

Sustituimos

$$ I=(-10)^2-4(7)(5)$$

$$ =100-140$$

$$ =-40$$

Como el indicador es negativo, debe tratarse de una elipse.

A) Elipse 

B) Hipérbola

C) Parábola

D) Recta

**Solución:** A

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la cónica correcta:

<img src="ej4_conica.png" align="middle" width="1000">

Finalmente, les dejo la dirección electrónica de mi gráfica de `Desmos` para que jueguen:

https://www.desmos.com/calculator/ghag3b70az